### PR TITLE
Restyle the notification bar

### PR DESCRIPTION
**Pull request by Gara64**
Change the design for the notification bar when a password can be added or updated.

See the final result:

Capture d’écran du 2019-11-18 18-03-35

### CODE REVIEW

File: src/notification/bar.scss [hunk 1]
```diff
@@ -16,72 +14,114 @@ table {
...
+.inner-table td button {
+    margin-left: 5px;
+}
 
-img {
-    border: 0;
-    margin: 0;
-    padding: 0;
+.desc-text {
+    color: #ffffff;
```
Comment (y-lohse) [inline src/notification/bar.scss]: Should we use a variable for this color too ?

File: src/notification/bar.scss [hunk 2]
```diff
@@ -16,72 +14,114 @@ table {
...
+    margin-top: 15px;
+    font-weight: 600;
```
Comment (y-lohse) [inline src/notification/bar.scss]: I always find numbered font weights weird. Is it `bold`?

File: src/notification/bar.scss [hunk 3]
```diff
@@ -16,72 +14,114 @@ table {
...
 }
 
-#close {
-    width: 15px;
-    height: 15px;
-    display: block;
-    padding: 5px 0;
-    margin-right: 10px;
+.more-options-label {
+    color: #aecdf6;
```
Comment (y-lohse) [inline src/notification/bar.scss]: This one should be a variable!

File: src/notification/bar.scss [hunk 4]
```diff
@@ -16,72 +14,114 @@ table {
...
 
-button:not(.link) {
-    background-color: $brand-primary;
-    padding: 5px 15px;
-    border-radius: 3px;
+.more-options-list {
+    margin-top: 15px;
     color: #ffffff;
-    border: 0;
+    background-color: #5397f4;
```
Comment (y-lohse) [inline src/notification/bar.scss]: Variable too? :)

File: src/notification/bar.scss [hunk 5]
```diff
@@ -16,72 +14,114 @@ table {
...
+    margin-top: 5px;
+}
+
+button {
+    border: 1px solid white;
+    font-weight: 600;
+    text-transform: uppercase;
+}
+
+button:not(.link) {
```
Comment (y-lohse) [inline src/notification/bar.scss]: Would it make sense to give it a specific class?

File: src/notification/bar.scss (hunk 2 above)
Comment (Gara64) [reply src/notification/bar.scss]: A bit less bold than bold (which is 700) :) No problem to use bold

File: src/notification/bar.scss (hunk 5 above)
Comment (Gara64) [reply src/notification/bar.scss]: Yeah I kept it as it was, but it probably makes more sense to use a class

**Pull request merged**